src/isomorphic/classic/types/ReactPropTypes.js

```diff
@@ -110,10 +111,21 @@ function createChainableTypeChecker(validate) {
     propName,
     componentName,
     location,
-    propFullName
+    propFullName,
+    secret
   ) {
     componentName = componentName || ANONYMOUS;
     propFullName = propFullName || propName;
+    if (!__DEV__) {
+      if (secret !== ReactPropTypesSecret && typeof console !== 'undefined') {
+        console.error(
```

Comment:
So should I use a generic message that doesn't mention the component name/prop then? I included that because I thought it would be useful, but if we're only warning once globally then it won't really matter